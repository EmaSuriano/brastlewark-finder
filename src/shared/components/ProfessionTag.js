import styled from 'styled-components';

const ProfessionTag = styled.span`
  border-radius: 2px;
  background-color: ${props => props.theme.accent}1A;
  border: 1px solid ${props => props.theme.accent}40;
  color: ${props => props.theme.secondary};
  display: inline-block;
  font-size: 0.9em;
  line-height: 1.4;
  margin-right: 5px;
  margin-top: 5px;
  padding: 3px;
`;

ProfessionTag.displayName = 'ProfessionTag';

export default ProfessionTag;
