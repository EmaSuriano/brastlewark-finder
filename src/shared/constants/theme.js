export default {
  background: '#4F5D75',
  light: '#FFFFFF',
  disabled: '#BFC0C0',
  accent: '#EF8354',
  secondary: '#2D3142',
};
