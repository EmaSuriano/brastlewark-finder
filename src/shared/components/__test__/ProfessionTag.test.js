import React from 'react';
import { shallow } from 'enzyme';
import theme from 'shared/constants/theme';
import ProfessionTag from '../ProfessionTag';

describe('<ProfessionTag />', () => {
  it('render', () => {
    expect(shallow(<ProfessionTag theme={theme} />)).toMatchSnapshot();
  });
});
