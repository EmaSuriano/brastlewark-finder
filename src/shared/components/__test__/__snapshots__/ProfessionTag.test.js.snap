// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfessionTag /> render 1`] = `
.c0 {
  border-radius: 2px;
  background-color: #EF83541A;
  border: 1px solid #EF835440;
  color: #2D3142;
  display: inline-block;
  font-size: 0.9em;
  line-height: 1.4;
  margin-right: 5px;
  margin-top: 5px;
  padding: 3px;
}

<span
  className="c0"
/>
`;
